cups/dest.c

```diff
@@ -40,7 +40,7 @@
 #  define kUseLastPrinter	CFSTR("UseLastPrinter")
 #endif // __APPLE__
 
-#define _CUPS_DNSSD_GET_DESTS	250	// Milliseconds for cupsGetDests
+#define _CUPS_DNSSD_GET_DESTS	1000	// Milliseconds for cupsGetDests
```

Comment:
@michaelrsweet suggested up to 10 sec (would be 10000) here. Would also not cause any problem and cover even more weird systems ...

Reply:
@tillkamppeter I suggested 10 seconds for `lpstat -e`, 1 second for `cupsGetDests` seems like a safer limit for applications, especially when `cupsEnumDests` is a better choice...

Reply:
@tillkamppeter there is 1 sec in the description so I used that for that reason as well :) .